src/utils.rs

```diff
@@ -0,0 +1,25 @@
+use std::{error::Error, fmt::Display};
+
+use napi::{bindgen_prelude::BigInt, Status};
+
+/// Convert rust error to napi::Error
+pub(crate) fn err_to_napi<T: Error>(e: T) -> napi::Error {
+    js_error(e)
+}
+
+/// Crete napi::Error error from a message
+pub(crate) fn js_error<T: Display>(e: T) -> napi::Error {
+    napi::Error::new(Status::GenericFailure, e.to_string())
+}
+
+// This is util function that will be used in #58 and #49 PR
+#[allow(dead_code)]
+pub(crate) fn bigint_to_i64<T: Display>(value: BigInt, error_msg: T) -> napi::Result<i64> {
+    let (i64_value, is_lossless) = value.get_i64();
+    // If i64 is negative, this means we would loose first bit of information,
+    // as value is converted from u64 without checking sign bit
```

Comment:
Imo ``this means`` is not needed here and ``checking the sign bit`` would sound better.